example_configs/ejabberd.md

```diff
@@ -0,0 +1,30 @@
+# Basic LDAP auth for a Ejabberd XMPP server
+
+[Main documentation here.](https://docs.ejabberd.im/admin/configuration/ldap/)
+
+For simple user auth add this to main ejabberd.yml:
+
+```
+host_config:
+  xmpp.example.org:
+    auth_method: [ldap]
+    ldap_servers:
+      - 127.0.0.1
+    ldap_port: 3890
+    ldap_uids:
+      - uid
+    ldap_rootdn: "uid=admin,ou=people,dc=example,dc=org"
```

Comment:
You could mention that any member of "lldap_admin" or "lldap_readonly" would work here (that still allows to change passwords)